demisto_sdk/commands/generate_outputs/generate_outputs_test.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from click.testing import CliRunner
+
+import demisto_sdk.__main__ as main
+
```

Comment:
correct me if I am wrong, but seems we do not test that real generate a regular short outputs and readme.

Reply:
It is done internally here: `demisto_sdk/commands/generate_context/generate_integration_context.py:79`